exporter/zipkinexporter/zipkin_test.go

```diff
@@ -30,8 +29,11 @@ import (
 
 	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
 	zipkinmodel "github.com/openzipkin/zipkin-go/model"
+	zipkinproto "github.com/openzipkin/zipkin-go/proto/v2"
 	zipkinreporter "github.com/openzipkin/zipkin-go/reporter"
 
```

Comment:
Please remove the empty line.
We use 3 import sections: standard, 3rd party, our and only use empty lines between sections.